hudi-common/src/main/java/org/apache/hudi/common/util/CompactionUtils.java

```diff
@@ -279,8 +279,8 @@ public static List<HoodieInstant> getPendingCompactionInstantTimes(HoodieTableMe
    */
   public static Option<Pair<HoodieTimeline, HoodieInstant>> getDeltaCommitsSinceLatestCompaction(
       HoodieActiveTimeline activeTimeline) {
-    Option<HoodieInstant> lastCompaction = activeTimeline.getCommitTimeline()
-        .filterCompletedInstants().lastInstant();
+    Option<HoodieInstant> lastCompaction = activeTimeline.getCommitAndCompactionTimeline()
+        .lastInstant();
```

Comment:
We should not change this, which is used for `CompactionTriggerStrategy.NUM_COMMITS`.  Otherwise, the semantics would change.